### ISSUE TITLE
numbered buttons are not working

### ISSUE THREAD

**Issue by Anshgrover23** [vishwamartur/Realstate#6]
#### Referenced screenshot:

![image](https://github.com/user-attachments/assets/3d70afe4-4447-4d94-bfd9-21e1c8423955)

**Comment by vishwamartur** [comment]
@Anshgrover23  
Sure i will check on it.

**Comment by Anshgrover23** [comment]
@vishwamartur ok bro. you are using too much of AI, have u made this website on your own.

**Comment by vishwamartur** [comment]
how do you verify stripe payment? In India, it is not working.

@Anshgrover23

**Comment by Anshgrover23** [comment]
@vishwamartur are you talking about algora stripe payments?

**Comment by vishwamartur** [comment]
@Anshgrover23 Yes bro and any international payments to receive Paypal are not preferred they asking for a stripe

**Comment by Anshgrover23** [comment]
Ask algora.io support regarding this, I have no idea brother.

**Comment by vishwamartur** [comment]
then how do you receive payments?

@Anshgrover23

**Comment by Anshgrover23** [comment]
@vishwamartur through stripe i said i did not knew about other things

**Comment by vishwamartur** [comment]
@Anshgrover23 How did you verified the stripe please share me to 
in india.

**Comment by Anshgrover23** [comment]
You only need to follow the steps of algora. Io i ha e just add my mobile number in stripe and add a bank account in that you do not to separatly make a account in stripe just follow as go in algora. Or if u have any other question regarding this just ask algora. Io help
